providers/os/resources/dockerfile.go

```diff
@@ -220,6 +222,7 @@ func (p *mqlDockerFile) stage2resource(stage instructions.Stage) (*mqlDockerFile
 				src[i] = v.SourcesAndDest.SourcePaths[i]
 			}
 			resource, err := CreateResource(p.MqlRuntime, "docker.file.add", map[string]*llx.RawData{
+				"__id":  llx.StringData(p.locationID(v.Location())),
```

Comment:
Why is this __id needed? Have we used such a thing before?

Reply:
This is only done because we don't have line context on the child resources yet, but definitely will add it. I'd love to pull the ID-generation in once we have line context back. I took a bit of a shortcut here :)